auction-server/api-types/src/opportunity.rs

```diff
@@ -206,18 +205,18 @@ pub enum OpportunityCreateProgramParamsV1Svm {
         #[serde_as(as = "DisplayFromStr")]
         order_address: Pubkey,
     },
-    /// Phantom program specific parameters for the opportunity.
-    #[serde(rename = "phantom")]
-    #[schema(title = "phantom")]
-    Phantom {
+    /// Swap program specific parameters for the opportunity.
+    #[serde(rename = "swap")]
+    #[schema(title = "swap")]
+    Swap {
         /// The user wallet address which requested the quote from the wallet.
         #[schema(example = "DUcTi3rDyS5QEmZ4BNRBejtArmDCWaPYGfN44vBJXKL5", value_type = String)]
         #[serde_as(as = "DisplayFromStr")]
         user_wallet_address: Pubkey,
 
-        /// The maximum slippage percentage that the user is willing to accept.
-        #[schema(example = 0.5, value_type = f64)]
-        maximum_slippage_percentage: f64,
+        /// The maximum slippage in basis points that the user is willing to accept.
+        #[schema(example = 50, value_type = u16)]
+        maximum_slippage_bps: u16,
```

Comment:
I think slippage doesn't make sense anymore in this context.

Reply:
Unless there's a ref price

Reply:
i agree, i kept it in here for legacy reasons, since wallets will always pass this number in their requests. we don't have to consume it, but we could also get rid of it.